src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Button = (name) => (
+  <button type="button">{name.btnName}</button>
```

Comment:
You should remove `.btnName`.
I don't understand where `btnName` is being added to your code